pkg/cli/mirror/helm.go

```diff
@@ -111,7 +125,48 @@ func (h *HelmOptions) PullCharts(ctx context.Context, cfg v1alpha2.ImageSetConfi
 
 	// Image download planning
 	additional := NewAdditionalOptions(h.MirrorOptions)
-	return additional.Plan(ctx, images)
+	var img []v1alpha2.Image
+	for _, image := range images {
+		img = append(img, v1alpha2.Image{Name: strings.ToLower(image.Name)})
+	}
+	return additional.Plan(ctx, img)
+}
+
+func IndexFile(indexURL string) ([]v1alpha2.Chart, error) {
+	var charts []v1alpha2.Chart
+	var indexFile helmrepo.IndexFile
+	httpClient := http.Client{Timeout: time.Duration(5) * time.Second}
+	if !strings.HasSuffix(indexURL, "/index.yaml") {
+		indexURL += "/index.yaml"
```

Comment:
in order to avoid the  `//` in the case that indexURL ends with a `/`, you can use things like `filepath.Join` , which would guarantee only 1 `/`

Reply:
When i am using the filepath.Join i get the url as https:/charts.openshift.io/index.yaml. It is resulting in a yaml parse error.